src/lambda/local/debugConfiguration.ts

```diff
@@ -3,19 +3,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import * as os from 'os'
-import * as vscode from 'vscode'
-import { DRIVE_LETTER_REGEX } from '../../shared/codelens/codeLensUtils'
+import { nodeJsRuntimes } from '../../lambda/models/samLambdaRuntime'
+import { SamLaunchRequestArgs } from '../../shared/sam/debugger/samDebugSession'
+import { dotNetRuntimes, pythonRuntimes, RuntimeFamily } from '../models/samLambdaRuntime'
 
-const DOTNET_CORE_DEBUGGER_PATH = '/tmp/lambci_debug_files/vsdbg'
+export const DOTNET_CORE_DEBUGGER_PATH = '/tmp/lambci_debug_files/vsdbg'
```

Comment:
Should this move to the cSharp debug configuration file instead of exporting?

Reply:
Having "debug config" stuff in one place is useful until it becomes a problem. When it becomes a problem that it's all in one place, then we should break it up. But not before then.